packages/cubejs-schema-compiler/src/adapter/BaseQuery.js

```diff
@@ -261,6 +261,8 @@ export class BaseQuery {
     }).filter(R.identity).map(this.newTimeDimension.bind(this));
     this.allFilters = this.timeDimensions.concat(this.segments).concat(this.filters);
 
+    this.collectAllMemberNames(); // Ensure member names are computed so that they are not lazily added during join computation
```

Comment:
This change is a bit unexpected. Is this something you really wanted to add to the fork and deviate from upstream?

I thought we just wanted to bump the versions